docs/tools/spark_block_cleaner.md

```diff
@@ -4,90 +4,105 @@
 
 </div>
 
-# Kubernetes tools spark-block-cleaner
+# Kubernetes Tools Spark Block Cleaner
 
 ## Requirements
 You'd better have cognition upon the following things when you want to use spark-block-cleaner.
 
 * Read this article
 * An active Kubernetes cluster
 * [Kubectl](https://kubernetes.io/docs/reference/kubectl/overview/)
+* [Docker](https://www.docker.com/)
 
 ## Purpose
+During using Spark On Kubernetes with client deploy-mode, we encountered this scenario that the disk responsible for storing Shuffle data accumulates so many files cause the disk overflows.
```

Comment:
Do not tell ‘our’ story, say it as a common case

Reply:
Ok，I will make it common